apps/skilavottord/README.md

```diff
@@ -32,6 +33,18 @@ See role description further down.
 
 ## Getting started
 
+### Setup
+
+Please fetch environment secret variables using the
+`yarn get-secrets skilavottord-ws` command.
+
+Additionally add the following environment variables to `.env.secret` in the
+project's root:
+
+```bash
+export NEXTAUTH_URL=http://localhost:4200/app/skilavottord/api/auth
```

Comment:
Is this needed? Seems like the default value in the environments is the same.

Reply:
next-auth requires this environment variable to be set.

Reply:
Huh, even though it is set here: [https://github.com/island-is/island.is/pull/6119/files#diff-907a615c9ccc2e81a8b2f74817e2c8c930e0fd17f481cac80379f51cef3a328cR9](https://github.com/island-is/island.is/pull/6119/files#diff-907a615c9ccc2e81a8b2f74817e2c8c930e0fd17f481cac80379f51cef3a328cR9)?

Reply:
yuup

Reply:
:exploding_head: